pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -260,6 +302,29 @@
 				</buttonbar>
 			</panel>
 
+			<tab><translate>Renderer</translate></tab>
+			<panel>
+				<h2><translate>Material system</translate></h2>
+				<row>
+					<h3><translate>Material system</translate></h3>
+					<input cvar="r_materialSystem" type="checkbox" />
+					<p><translate>Use a new system that significantly improves performance. Some drivers may fail with this on (e. g. Mesa radeonsi and AMD proprietary drivers).</translate>&nbsp;<translate>Experimental.</translate></p>
```

Comment:
You can just say “Mesa” instead of “Mesa radeonsi”. I was able to test the code on Mesa zink too (OpenGL implementation over Vulkan), and the culling code fails like the radeonsi one, and the Mesa code where the breakage occurs is in some shared gallium code.

So material system works on Mesa with the latest fixes after our previous reports, but culling hits some other bugs in the common Mesa code as well.

Reply:
Ah, I see. Is it only the occlusion culling that crashes on Mesa?

Reply:
Alright, I changed this to just say "Mesa".